README.md

```diff
@@ -1,7 +1,150 @@
+The XxllncZGWBundle is a Symfony bundle that provides functionality for the CommonGateway about handling synchronizations between the xxllnc zaaksysteem standard and the ZGW standard. So it fetches and send xxllnc objects from and to their ZGW object equilevants.
 
-Codacy rating [![Codacy Badge](https://app.codacy.com/project/badge/Grade/636ff2fbbcbd423dab24940ec99ad19e)](https://www.codacy.com/gh/CommonGateway/XxllncZGWBundle/dashboard?utm_source=github.com\&utm_medium=referral\&utm_content=CommonGateway/XxllncZGWBundle\&utm_campaign=Badge_Grade)
+For more information on ZaakGerichtWerken and XXLNC, please visit [here]([https://github.com/vrijBRP/vrijBRP]\(https://xxllnc.nl/zaakgericht\)https://xxllnc.nl/zaakgericht)
 
-This plugin provides `ZGW` functionality for the XXLLNC API. For More information on ZaakGerichtWerken and XXLNC, please visit [here]([https://github.com/vrijBRP/vrijBRP]\(https://xxllnc.nl/zaakgericht\)https://xxllnc.nl/zaakgericht)
+### Bundles Used in the XxllncZGWBundle
+
+The [**XxllncZGWBundle**](https://github.com/CommonGateway/XxllncZGWBundle) currently utilizes the following bundles:
+
+1.  **CoreBundle**: [GitHub Repository](https://github.com/CommonGateway/CoreBundle)
+2.  **ZGWBundle**: [GitHub Repository](https://github.com/CommonGateway/BRPBundle) (the ZGWBundle also includes other bundles like the KlantenBundle which is part of ZGW.)
+
+While the ZGW and CoreBundle can still be installed as standalone components (please refer to their respective installation guides), the XxllncZGWBundle now defaults to installing these bundles as additional plugins on the same gateway.
+
+## Backend Installation Instructions
+
+The XxllncZGWBundle backend codebase utilizes the Common Gateway as an open-source installation framework. This means that the XxllncZGWBundle library, in its core form, functions as a plugin on this Framework. To learn more about the Common Gateway, you can refer to the documentation [here](https://commongateway.readthedocs.io/en/latest/).
+
+The CommonGateway Admin UI frontend comes with te CommonGateway.
+
+To install the backend, follow the steps below:
+
+### Gateway Installation
+
+1.  If you do not have the Common Gateway installed, you can follow the installation guide provided [here](https://github.com/ConductionNL/commonground-gateway#readme). The Common Gateway installation is required for the backend setup. You can choose any installation method for the gateway, such as Haven, Kubernetes, Linux, or Azure, and any database option like MySQL, PostgreSQL, Oracle, or MsSQL. The gateway framework handles this abstraction.
+
+### XxllncZGWBundle Installation - Admin-UI
```

Comment:
A little while ago we (Robert, Sarai, and I) decided to call it the Gateway UI.
I think it is best to be consistent about this and call it "Gateway UI" everywhere, haven't had the chance to ask Ruben his opinion on this however